Sources/CoreDataRepository/NSManagedObjectContext+Helpers.swift

```diff
@@ -11,15 +11,15 @@ import Foundation
 
 extension NSManagedObjectContext {
     /// Helper function for getting the ``NSManagedObjectID`` from an ``URL``
-    func objectId(from url: URL) -> Result<NSManagedObjectID, CoreDataError> {
+    public func objectId(from url: URL) -> Result<NSManagedObjectID, CoreDataError> {
         guard let objectId = persistentStoreCoordinator?.managedObjectID(forURIRepresentation: url) else {
             return .failure(CoreDataError.failedToGetObjectIdFromUrl(url))
         }
         return .success(objectId)
     }
 
     /// Helper function for checking that a managed object is not deleted in the store
-    func notDeletedObject(for id: NSManagedObjectID) throws -> NSManagedObject {
+    public func notDeletedObject(for id: NSManagedObjectID) throws -> NSManagedObject {
```

Comment:
What was the reason this needed to be made public?

Reply:
It's a method that would likely be helpful for use in the new `custom` method